lib/std/sha1.nim

```diff
@@ -8,6 +8,32 @@
 #
 
 ## Note: Import ``std/sha1`` to use this module
```

Comment:
I know it's not yours but bold this please

Reply:
Line made bold: 
```nim
## **Note: Import `std/sha1` to use this module**
```